app/scripts/modules/core/src/pipeline/config/triggers/Triggers.tsx

```diff
@@ -15,15 +15,15 @@ export interface ITriggersProps {
   pipeline: IPipeline;
   fieldUpdated: () => void;
   updatePipelineConfig: (changes: Partial<IPipeline>) => void;
-  viewState: { revertCount: number };
+  revertCount: number;
```

Comment:
here's where we changed to a primitive value so the react component will get re-rendered by react2angular.